MapboxMobileEvents/MMEEventsManager.m

```diff
@@ -308,6 +315,35 @@ - (void)sendTurnstileEvent {
     }];
 }
 
+- (void)sendTelemetryMetricsEvent {
```

Comment:
this method is a bit hard to read. is there a way to split it into the multiple functions?

Reply:
Will split this into a method for generating the `MMEEvent`